src/main/webapp/scripts/appscript.js

```diff
@@ -22,18 +22,6 @@ async function populateCustomerList() {
 
   // Response is a json array that is parsed into json.
   var customerJson = await response.json();
-
```

Comment:
looks like you may have a merge  conflict here with Vincent's latest PR. not sure